go.mod

```diff
@@ -4,16 +4,16 @@ go 1.18
 
 require (
 	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869
-	github.com/go-logr/logr v1.2.3
+	github.com/go-logr/logr v1.2.4
 	github.com/google/go-cmp v0.5.9
 	github.com/jenkins-x/go-scm v1.10.10
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/openshift/api v0.0.0-20210503193030-25175d9d392d
 	github.com/spf13/afero v1.8.0
-	github.com/stretchr/testify v1.8.0
+	github.com/stretchr/testify v1.8.2
 	go.uber.org/zap v1.24.0
-	k8s.io/api v0.26.1
-	k8s.io/apimachinery v0.26.1
+	k8s.io/api v0.28.3
+	k8s.io/apimachinery v0.28.3
```

Comment:
I think you can just bump these two up to `v0.26.10` unless there is a better reason to move up to `v0.28.x`.  
https://github.com/kubernetes/api/releases/tag/v0.26.10

Reply:
FYI I need to back track changes made on https://github.com/redhat-developer/gitops-generator/pull/51 where I updated to v0.28.3

Because importing the gitops generator in HAS, HAS has to also upgrade k8s.io/client-go and that means also upgrading controller-runtime in HAS otherwise I face assertion errors in controller-runtime

Seems like we have to settle with https://github.com/kubernetes/api/releases/tag/v0.26.10 until we can actually move controller-runtime up

Reply:
upgraded to only v0.26.10 and confirmed it builds and works with HAS controllers.